src/crud/document.py

```
@@ -578,7 +578,7 @@ async def create_documents(
 
                 except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)